### ISSUE TITLE
Timing in VAE+INN

### ISSUE THREAD

**Issue by fa7sa** [heidelberg-hepml/CaloINN#4]
While trying to generate samples, I noticed that it does not compute timing information. I am assuming that is same as previous CaloINN repo. However, I just want to make sure that I am calculating time properly as you did in the CaloINN paper. So, I am considering both latent sample creation and final sample creation time from VAE for total time for sampling one event. Is this correct? I am looking at the 'generate' function in trainer.py.

**Comment by floern1** [comment]
Yes, for the 2 stage approach witht the VAE, you should sum up both generation times. It should be safe to just time the trainer.generate function. For the VAE approach the postprocessing is not part of the generate function. So, you could just take the time and divide it by the number of created samples.
